spec/interactors/find_user_by_reset_token_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe FindUserByResetToken do
+  describe ".call" do
+    let(:token) { Encryptor.generate_token }
+
+    context "when successful" do
+      before do
+        @user = create(:confirmed_user, reset_digest: token[0])
+      end
```

Comment:
Hmm I see what you mean about not concerning ourselves and allocating responsibility to `find_by`. 

Even though ActiveRecord is responsible here, I still have to test the line anyway and I tend to think being more explicit in my expectation is better? Though maybe it's better to really be extra sure not to test things that are already thoroughly tested on their own. That way your tests cover very strictly only your contributions. I'm conflicted.